content/content-delivery/v1/core-resources/collaborators/add-collaborator.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Add a Collaborator
```

Comment:
You can't add collaborators from the CDN API

Reply:
You can add a collaborator or invite a collaborator using MAPI

Reply:
Yeah, but this is the folder of the CDN API :-)